src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingStegTest.kt

```diff
@@ -169,8 +195,21 @@ class VilkårsvurderingStegTest {
     }
 
     @Test
-    fun `utførSteg - skal validere vilkårsvurderingen`() {
+    fun `utførSteg - skal kaste funksjonell feil hvis ikke er noe barn i personopplysningsgrunnlaget`() {
         val barn = randomAktør()
+        val søknadGrunnlagMock = mockk<SøknadGrunnlag>(relaxed = true)
+
+        mockkObject(SøknadGrunnlagMapper)
+        with(SøknadGrunnlagMapper) {
+            every { søknadGrunnlagMock.tilSøknadDto() } returns SøknadDto(
+                søkerMedOpplysninger = SøkerMedOpplysningerDto("søkerIdent"),
+                barnaMedOpplysninger = listOf(
+                    BarnMedOpplysningerDto(ident = "barn1"),
+                    BarnMedOpplysningerDto("barn2")
```

Comment:
Vurdere å fjerne barna herfra eller spesifisere at de ikke er inkludert i søknaden for å tydeliggjøre hva vi tester her 😊